<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.11.9 on win32)" generated="2025-01-12T11:33:22.314779" rpa="false" schemaversion="5">
<suite id="s1" name="RobotTests" source="D:\Android\AndroidStudioProjects\MySimpleFirebaseProject\RobotTests">
<suite id="s1-s1" name="Test Suite" source="D:\Android\AndroidStudioProjects\MySimpleFirebaseProject\RobotTests\test_suite.robot">
<test id="s1-s1-t1" name="Test Cat Fact Fetch" line="12">
<kw name="Open Application" owner="AppiumLibrary">
<kw name="Capture Page Screenshot" owner="AppiumLibrary">
<msg time="2025-01-12T11:33:24.729363" level="FAIL">No application is open</msg>
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<status status="FAIL" start="2025-01-12T11:33:24.729363" elapsed="0.003011">No application is open</status>
</kw>
<msg time="2025-01-12T11:33:24.733389" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg time="2025-01-12T11:33:24.734367" level="FAIL">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</msg>
<arg>http://localhost:4723/wd/hub</arg>
<arg>platformName=${PLATFORM}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>app=${APP}</arg>
<arg>appPackage=${APP_PACKAGE}</arg>
<arg>appActivity=${APP_ACTIVITY}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://appium.io/docs/en/2.1/cli/args/
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<status status="FAIL" start="2025-01-12T11:33:22.540794" elapsed="2.194596">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</status>
</kw>
<kw name="Click Element" owner="AppiumLibrary">
<arg>id=buttonClick</arg>
<doc>Click element identified by `locator`.</doc>
<status status="NOT RUN" start="2025-01-12T11:33:24.735390" elapsed="0.000000"/>
</kw>
<kw name="Sleep" owner="BuiltIn">
<arg>5s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" start="2025-01-12T11:33:24.736364" elapsed="0.000000"/>
</kw>
<kw name="Get Text" owner="AppiumLibrary">
<var>${fact}</var>
<arg>id=catFactTextView</arg>
<doc>Get element text (for hybrid and mobile browser use `xpath` locator, others might cause problem)</doc>
<status status="NOT RUN" start="2025-01-12T11:33:24.736364" elapsed="0.000000"/>
</kw>
<kw name="Log" owner="BuiltIn">
<arg>Cat Fact: ${fact}</arg>
<doc>Logs the given message with the given level.</doc>
<status status="NOT RUN" start="2025-01-12T11:33:24.736364" elapsed="0.000000"/>
</kw>
<kw name="Close Application" owner="AppiumLibrary">
<doc>Closes the current application and also close webdriver session.</doc>
<status status="NOT RUN" start="2025-01-12T11:33:24.737360" elapsed="0.000000"/>
</kw>
<doc>Test fetching a cat fact and displaying it</doc>
<status status="FAIL" start="2025-01-12T11:33:22.538756" elapsed="2.198604">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</status>
</test>
<status status="FAIL" start="2025-01-12T11:33:22.352780" elapsed="2.386582"/>
</suite>
<status status="FAIL" start="2025-01-12T11:33:22.316758" elapsed="2.425604"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="RobotTests">RobotTests</stat>
<stat pass="0" fail="1" skip="0" id="s1-s1" name="Test Suite">RobotTests.Test Suite</stat>
</suite>
</statistics>
<errors>
<msg time="2025-01-12T11:33:24.733389" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
